src/components/map/ha-map.ts

```diff
@@ -555,6 +593,7 @@ export class HaMap extends ReactiveElement {
         color: white !important;
         border-radius: 4px;
         box-shadow: none !important;
+        text-align: center;
```

Comment:
Suggest to add breaking options since a name may be long

Reply:
Do you have a specific suggestion?

Reply:
Also add this to break whitespaces:
```
white-space: pre-line;
```

Reply:
Hm that's not doing anything. 

I think it's fine as-is, even name twice as long as you suggest looks OK on single line. The tooltip grows horizontally to accomodate, so it doesn't overflow anything.

Reply:
Cannot test it myself; so you can make it w/o breaking; will test & add it later when this will be merged.

Reply:
You probably do not see a breaking since the width is not limited.